dump-and-save.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# dump-and-save.sh - Dump Terraform Cloud workspaces and variables then save them to 1Password
+#
+# Dale Newby
+# SIL International
+# July 7, 2023
```

Comment:
I'm curious about the purpose of these three lines (name, org, date). It seems redundant with git commit history.

Reply:
Habit.  It answers the question "Who wrote this and when?" when a user finds the file and doesn't know from whence it came.